syncd/syncd_flex_counter.cpp

```diff
@@ -1017,7 +1017,7 @@ void FlexCounter::saiUpdateSupportedPortCounters(sai_object_id_t portId)
     SWSS_LOG_ENTER();
 
     uint64_t value;
-    for (int cntr_id = SAI_PORT_STAT_IF_IN_OCTETS; cntr_id <= SAI_PORT_STAT_PFC_7_ON2OFF_RX_PKTS; ++cntr_id)
+    for (int cntr_id = SAI_PORT_STAT_IF_IN_OCTETS; cntr_id <= SAI_PORT_STAT_PFC_7_TX_PAUSE_DURATION_US; ++cntr_id)
```

Comment:
why not use sai_metadata_enum_sai_bridge_port_stat_t .values and .values count to have all counters here regardless SAI update, then we can just forget about keeping this up to date??

Reply:
This ask is Nice-to-have